release-notes/opensearch-anomaly-detection-dashboards.release-notes-2.14.0.0.md

```diff
@@ -0,0 +1,24 @@
+## Version 2.14.0.0 Release Notes
+
+Compatible with OpenSearch Dashboards 2.14.0
+
+### Enhancements
+
+* Support MDS on List, Detail, Dashboard, Overview pages ([#722](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/722))
+
+### Bug Fixes
+
+* Populate selected indices from query params on initial load ([#713](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/713))
+* Clean selected field values when switching data source on create page ([#742](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/742))
```

Comment:
This is part of MDS support change, not an existing bug. Could you please remove it from this section please?